### PR TITLE
Feature/fix readmes

### PR DESCRIPTION
**Pull request by anwar6953**

### CODE REVIEW

File: cdap-stream-clients/python/README.md [hunk 1]
```diff
@@ -22,120 +29,102 @@ Stream Client Python API for managing Streams via external Python applications.
...
-    from cdap_stream_client import StreamClient
+  from cdap_stream_client import Config
+  from cdap_stream_client import StreamClient
 ```
 
 ## Example
+#### Create StreamClient
+Create a ```StreamClient``` instance, specifying the fields 'host' and 'port' of the gateway server.
+```
+   stream_client = StreamClient()
```
Comment (poornachandra) [inline cdap-stream-clients/python/README.md:42]: The sample code does not have host and port set.

File: cdap-stream-clients/python/README.md [hunk 2]
```diff
@@ -22,120 +29,102 @@ Stream Client Python API for managing Streams via external Python applications.
...
 ```
-    from cdap_stream_client import Config
-    from cdap_stream_client import StreamClient
+  from cdap_stream_client import Config
+  from cdap_stream_client import StreamClient
 ```
 
 ## Example
+#### Create StreamClient
+Create a ```StreamClient``` instance, specifying the fields 'host' and 'port' of the gateway server.
```
Comment (poornachandra) [inline cdap-stream-clients/python/README.md]: of the gateway server => of the CDAP instance

File: cdap-stream-clients/python/README.md [hunk 3]
```diff
@@ -1,20 +1,28 @@
...
+ - Create a Stream
+ - Update TTL (time-to-live) for an existing Stream
+ - Retrieve the current Stream TTL
+ - Truncate an existing Stream (the deletion of all events that were written to the Stream)
+ - Write an event to an existing Stream
 
 
 ## Installation
- To install CDAP Authentication Client, run:
+ To install CDAP Authentication Client, either [download a zip file](https://repository.cask.co/downloads/co/cask/cdap/cdap-python-stream-client/1.0.0/cdap-python-stream-client-1.0.0.zip)
```
Comment (awholegunch) [inline cdap-stream-clients/python/README.md]: Start "(https..." on a new line.

Comment (awholegunch) [inline cdap-stream-clients/python/README.md]: To install the CDAP ...

File: cdap-stream-clients/python/README.md [hunk 4]
```diff
@@ -1,20 +1,28 @@
...
+ 
+ ```
+ $ unzip cdap-python-authentication-client-1.0.0.zip
+ $ cd cdap-python-authentication-client-1.0.0
+ $ python setup.py install
+ ```
+ 
+ or [clone the repository](https://github.com/caskdata/cdap-ingest)
```
Comment (awholegunch) [inline cdap-stream-clients/python/README.md:22]: Are these links correct? I'm confused...stream-client? authentication-client?

File: cdap-stream-clients/python/README.md [hunk 5]
```diff
@@ -1,20 +1,28 @@
...
 ```
-    $ python setup.py install
+ $ git clone https://github.com/caskdata/cdap-clients.git
```
Comment (awholegunch) [inline cdap-stream-clients/python/README.md]: cdap-ingest.git ?

File: cdap-stream-clients/python/README.md (hunk 4 above)
Comment (anwar6953) [inline cdap-stream-clients/python/README.md:22]: They're wrong, will fix.

File: cdap-stream-clients/python/README.md [hunk 6]
```diff
@@ -1,20 +1,29 @@
...
+ - Update TTL (time-to-live) for an existing Stream
+ - Retrieve the current Stream TTL
+ - Truncate an existing Stream (the deletion of all events that were written to the Stream)
+ - Write an event to an existing Stream
 
 
 ## Installation
- To install CDAP Authentication Client, run:
+ To install the CDAP Stream Client, either [download a zip file]
+ (https://repository.cask.co/downloads/co/cask/cdap/cdap-python-stream-client/1.0.0/cdap-python-stream-client-1.0.0.zip)
```
Comment (poornachandra) [inline cdap-stream-clients/python/README.md:14]: This should be 1.0.1. Should we make another pass after the artifacts are built to update the links?

File: cdap-stream-clients/python/README.md [hunk 7]
```diff
@@ -1,20 +1,29 @@
...
+ 
+ ```
+ $ unzip cdap-python-stream-client-1.0.0.zip
```
Comment (poornachandra) [inline cdap-stream-clients/python/README.md:17]: 1.0.1 here too.

**Comment by poornachandra** [comment]
LGTM :+1:  

We will have to make version same changes here, in auth client readme and java versions too. We can do these changes in a separate PR.

**Comment by awholegunch** [comment]
LGTM.

**Pull request merged**